editor/editor_properties.cpp

```diff
@@ -2191,7 +2187,6 @@ EditorPropertyColor::EditorPropertyColor() {
 	add_child(picker);
 	picker->set_flat(true);
 	picker->connect("color_changed", callable_mp(this, &EditorPropertyColor::_color_changed));
-	picker->connect("popup_closed", callable_mp(this, &EditorPropertyColor::_popup_closed));
```

Comment:
There's now nothing connected to the `popup_closed` signal.

It used to be emitted when receiving `modal_closed`, which happens when the ColorPicker popup receives `NOTIFICATION_MODAL_CLOSE`. Are you sure that this doesn't trigger in situations where you might still want to validate the input?

Reply:
> Are you sure that this doesn't trigger in situations where you might still want to validate the input?

That could only happen when you modify the color without triggering `_color_changed()` and then close the popup. AFAIK it's not possible.